src/common/meta/src/wal/kafka/topic_manager.rs

```diff
@@ -141,6 +154,31 @@ impl TopicManager {
             .collect()
     }
 
+    async fn try_create_noop_record(&self, topic: &Topic, client: &Client) -> Result<()> {
+        let partition_client = client
+            .partition_client(topic, DEFAULT_PARTITION, UnknownTopicHandling::Retry)
+            .await
+            .context(BuildKafkaPartitionClientSnafu {
+                topic,
+                partition: DEFAULT_PARTITION,
+            })?;
+
+        partition_client
+            .produce(
+                vec![Record {
+                    key: None,
+                    value: None,
+                    timestamp: rskafka::chrono::Utc::now(),
```

Comment:
There may be a precision issue incurred by `Utc::now`. You can write a demo which produces a record timestamped with `Utc::now` and fetches the record. The produced one won't be identical with the fetched one.

Reply:
For starting a Kafka test environment, I suggest to use https://github.com/conduktor/kafka-stack-docker-compose which provides many easy-to-use scripts to start various Kafka clusters with a single command.

Reply:
> There may be a precision issue incurred by `Utc::now`. You can write a demo which produces a record timestamped with `Utc::now` and fetches the record. The produced one won't be identical with the fetched one.

Okay, do you have any alternative? I just copied from the `record_utils.rs`.

Reply:
> For starting a Kafka test environment, I suggest to use https://github.com/conduktor/kafka-stack-docker-compose which provides many easy-to-use scripts to start various Kafka clusters with a single command.

It doesn't matter now. And we can easily switch to that solution in the future. In fact, I wouldn't say I like any solution with the zookeeper; it's really heavy and takes me back to 10 years ago. Let's use the kraft and move quickly.

Reply:
Agreed with the opinion on zookeeper. 
For the timestamp, we need to find a way to handle the precision issue or it may affect testing.

Reply:
Do we rely on it?

Reply:
It seems does not matter for now.